### ISSUE TITLE
[ENH] Node title as tooltip

### ISSUE THREAD

**Issue by thriveth** [org-roam/org-roam-ui#175]
**Is your feature request related to a problem? Please describe.**
When the graph is zoomed out enough that labels are not showing, I would still like to be able to orient myself quickly without having to zoom in and out.

**Describe the solution you'd like**
- Show node title in a tooltip on mouse hover. 
- Possibly disable this when the label is already showing at above a certain size.

**Comment by ThomasFKJorna** [comment]
Good point, I actually removed the default tool tip with the intention of adding it back in some form later, but never got around to it! I'll re-add it

**Comment by thriveth** [comment]
Great! It is one of those little things I find myself missing many times every day.
